### PR TITLE
refactor: rename Member.APIEndpoint field

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Review by ranakan19 — APPROVED** [review]
LGTM

**Review by ranakan19 — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -29,4 +29,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220127135604-0fab356f7a15
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20220127135755-5e21549c9cb8
```
Comment (sbryzak) [inline go.mod:34]: Need to remove these

**Review by sbryzak — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (xcoulon) [reply go.mod:34]: yes, I'll do it as soon as the related PRs are merged

Comment (xcoulon) [reply go.mod]: removed in 6b610e2

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**